docs/3.0-migration-guide.d/turn_SSL_SRV_RESPECT_CLIENT_PREFERENCE_config_opt_to_runtime_opt.md

```diff
@@ -0,0 +1,13 @@
+Turn MBEDTLS_SSL_SRV_RESPECT_CLIENT_PREFERENCE configuration option into a runtime option
+--
+
+This change affects users who see the change of the SSL server vs. client
+preferred set of ciphersuites in runtime useful.
+
+The `MBEDTLS_SSL_SRV_RESPECT_CLIENT_PREFERENCE` `config.h` option has been
+removed and a new function with similar functionality has been introduced into the
+SSL API.
+
+This new function `mbedtls_ssl_conf_respect_client_preference()` can be used to
+change the preferred set of ciphersuites on the server to those used on the client.
+The default state is to use the server set of suites.
```

Comment:
Concretely: if you were enabling `MBEDTLS_SSL_SRV_RESPECT_CLIENT_PREFERENCE` at compile time, you should now call this when configuring SSL connections:
```
mbedtls_ssl_conf_respect_client_preference(ssl_config, MBEDTLS_SSL_SRV_RESPECT_CLIENT_PREF_ENABLED);
```